sdk/Javascript/src/Push/Push.FxOS.js

```diff
@@ -0,0 +1,413 @@
+// ----------------------------------------------------------------------------
```

Comment:
In first, I tried to modify Web.js. However parent window object don't access to child window object and  on FirefoxOS app.
Using iframe too.
In result, it need to use MozBrowser (It is like webview on Android app) to work single sign-on. 
Therefore I create new file.